package.json

```diff
@@ -33,6 +33,7 @@
     "concurrently": "^5.3.0",
```

Comment:
This is cool! For running multiple scripts in the same terminal?

Reply:
Yep. I'm not using it currently, but yeah it can be helpful to write one script that runs your webpack watch, starts up your express server, etc.